### PR TITLE
Tests for owner

### PR DESCRIPTION
**Pull request by tarasiki**
tests cl1-cl5

### CODE REVIEW

File: EasyRestProjectNetTeam2/EasyRestPages/MenuPage.cs [hunk 1]
```diff
@@ -6,8 +6,8 @@ namespace EasyRestProjectNetTeam2.EasyRestPages
 {
     public class MenuPage : BasePage
     {
-        public DatePickerComponent DatePickerComponent { get; }
-        public TimePickerComponent TimePickerComponent { get; }
+        public DatePickerComponent DatePickerComponent { get; set; }
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/MenuPage.cs]: why did you add public setters?

Comment (tarasiki) [reply EasyRestProjectNetTeam2/EasyRestPages/MenuPage.cs]: because we caught errors

Comment (dmatsev) [reply EasyRestProjectNetTeam2/EasyRestPages/MenuPage.cs]: you can't have a public setter because you can re-initialize the component on the test level at any moment, please change the access modifier

File: EasyRestProjectNetTeam2/EasyRestPages/SignUpPage.cs [hunk 2]
```diff
@@ -6,7 +6,7 @@ namespace EasyRestProjectNetTeam2.EasyRestPages
 {
     public class SignUpPage : BasePage
     {
-        public DatePickerComponent DatePickerComponent { get; }
+        public DatePickerComponent DatePickerComponent { get; set; }
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/SignUpPage.cs]: please change the access modifier for setter

**Review by AhnessP — APPROVED** [review]

File: EasyRestProjectNetTeam2/EasyRestTests/CheckRestaurantsInfoTests.cs [hunk 3]
```diff
@@ -1,13 +1,80 @@
...
+        {
+            base.SetUp();
+            homePage = GetHomePage();
+            homePage.HeaderMenuComponent.ClickSignInButton();
+            signInPage = GetSignInPage();
+            signInPage.SendKeysToInputEmail(dataModel.EmailForClient);
+            signInPage.SendKeysToInputPassword(dataModel.PasswordForClient);
+            homePage.HeaderMenuComponent.ClickSignInButton();        
+        }
+            [Test]
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestTests/CheckRestaurantsInfoTests.cs]: please format

File: EasyRestProjectNetTeam2/EasyRestTests/CheckRestaurantsInfoTests.cs [hunk 4]
```diff
@@ -1,13 +1,80 @@
...
+        public void CheckReastaurantListPagetest()
+        {
+            restaurantsPage = GetRestaurantsPage();
+            restaurantsPage.WaitAndClickResturantList(dataModel.TimeToWait);
+            var expectedUrl = dataModel.RestaurantsListPageUrl;
+            var actualUrl = restaurantsPage.GetPageUrl();
+            StringAssert.Contains(expectedUrl, actualUrl, "Search word is absent for menu URL");
+        }
+
+            [Test]
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestTests/CheckRestaurantsInfoTests.cs]: please format

**Review by dmatsev — APPROVED** [review]
Approve with comment
